### PR TITLE
fixed JWT authentication port

### PR DESCRIPTION
**Pull request by Giulio2002**

### CODE REVIEW

File: cmd/rpcdaemon/cli/config.go [hunk 1]
```diff
@@ -53,7 +54,7 @@ var rootCmd = &cobra.Command{
 }
 
 const JwtTokenExpiry = 5 * time.Second
-const JwtDefaultFile = "jwt.hex"
+const JwtDefaultFile = "/tmp/jwt.hex"
```
Comment (yperbasis) [inline cmd/rpcdaemon/cli/config.go:57]: Should use `os.TempDir()` instead of `/tmp`, at least in order to be compatible with Windows.

Comment (yperbasis) [reply cmd/rpcdaemon/cli/config.go:57]: Actually, I think it's better to follow geth's example and place this file inside datadir.

Comment (Giulio2002) [reply cmd/rpcdaemon/cli/config.go:57]: what if we do not specify any datadir in rpcdaemon, or what if we run the daemon on another machine than erigon?

Comment (Giulio2002) [reply cmd/rpcdaemon/cli/config.go:57]: I suggest to use current directory

Comment (yperbasis) [reply cmd/rpcdaemon/cli/config.go]: OK, let's stick with the current directory.

File: cmd/rpcdaemon/cli/config.go [hunk 2]
```diff
@@ -517,35 +518,48 @@ func isWebsocket(r *http.Request) bool {
...
+			return jwtSecret, nil
+		}
+		log.Error("Invalid JWT secret", "path", fileName, "length", len(jwtSecret))
+		return nil, errors.New("invalid JWT secret")
+	}
+	// Need to generate one
+	jwtSecret := make([]byte, 32)
+	rand.Read(jwtSecret)
+	// if we're in --dev mode, don't bother saving, just show it
+	if fileName == "" {
```
Comment (yperbasis) [inline cmd/rpcdaemon/cli/config.go:547]: This cannot happen because fileName is set to `JwtDefaultFile` if empty.

**Review by yperbasis — APPROVED** [review]

**Pull request merged**